### ISSUE TITLE
v1.

### ISSUE THREAD

**Issue by fititnt** [pokemongovet/pokemongo.vet.br#29]

**Comment by woliveiras** [comment]
Talvez se o App possuir mais coisas, a galera baixe e para adoção também.

Como eles, ao adotar, vão somente pesquisar, encontrar e depois não vão mais precisar do App, eles encaram como desnecessário a instalação ou mesmo instalam e depois removem.

Se tiver algo como Calendário de vacinas do seu Pet, Guia de alimentação, Dicas de adestramento e Diário do Pet (para ter registro de doenças que o animal possa ter durante a vida e ajudar a expor dados úteis para um veterinário, quando precisar ou mesmo algo legal para guardar fotos do bichin com compartilhamento em redes sociais), acho que o pessoal utilizaria mais o App.

**Comment by evansantos** [comment]
concordo contigo @woliveiras, o nível de adoção e nível de abandono podem ser vertiginosos depois da utilização do app, algo legal seria isso que você falou, mais no quesito de uma pokedex (~^•^~) e pensando na não necessidade de instalar nada no aparelho, uma abordagem legal seria fazer um (P)WA?

**Comment by icoffani** [comment]
Olá @woliveiras @evansantos e @carvalhobruno. Muito obrigada pelo interesse de vocês!

Um dos pontos-chave de quando criamos esse projeto era apenas pedir ajuda para coisas que, ou são triviais de serem feitas a ponto de correr o risco, ou ter um planejamento mínimo.

Esse issue aqui é uma ramificação do #27, que é uma coisa muito séria. Eu vejo que a gente tem programadores prontos pra fazer o que vier pela frente, mas os org admins da @pokemongovet estão precisando de muita ajuda justamente de não-programadores. Ou então de programadores que ajudem a fazer levantamento de requisitos e análise de mercado. 

Vou conversar com o pessoal aqui pra gente abrir issues focadas também nessas tarefas que não envolvem código, mas sem elas a gente não tem como ir adiante com mais segurança.

**Comment by icoffani** [comment]
Eu não tenho experiência tão grande quanto vocês aqui tem, mas eu estou ouvindo o pessoal pra tentar entender porque tantas boas ideias não vão adiante.

E uma coisa eu percebi: **quem se dedica demais em prol de um projeto open source por amor e não dinheiro, fica muito desanimado se ele não for usado**.  Por exemplo, [buscando por hackaton](https://github.com/search?q=hackaton) talvez a maior parte do que tem ali não deve ter sido usado no mundo real. [E por HackInPoa, um hackaton que ocorreu ano passado, nenhum deles parece ter virado um produto](https://github.com/search?q=hackinpoa).

Creio que vale à pena proteger não só quem é mais novo e quer dar o primeiro PR, mas também se formos abrir outro repositório que exija muito mais, que isso tenha condições de ir para frente e aprenda com erros de projetos anteriores.

Eu fico animada de vocês fazerem tantas sugestões, até porque podem falar de coisas que não pensamos! Porém, eu sinto que pra transformar o código que vocês dedicam aqui em mais do que uma notícia de jornal ou um emprego melhor por um head hunter, realmente seja convertido em resultado nos animais que estão abandonados.

**Comment by woliveiras** [comment]
Muito bem @icoffani!
